labeling/codebase/localcodebase.go

```diff
@@ -103,3 +103,16 @@ func (c LocalCodebase) FindFile(glob ...string) (path string, err error) {
 func (c LocalCodebase) ReadFile(path string) (contents []byte, err error) {
 	return os.ReadFile(filepath.Join(c.BasePath, path))
 }
+
+func (c LocalCodebase) ListFiles() ([]string, error) {
```

Comment:
I'm not clear why this is needed . What's the advantage over adding a check for `readme.md` in the existing loop where we filter out `.` ?

Reply:
IMO I think the concept of determining what an `empty` repo is belong to the labeling more than the codebase. For example it is possible that the rules change to include more than just the `readme.md` in the future (ex. .gitignore ). So I felt it would be better to have a function that just returns the list of files in the codebase and let the labelling rules figure what to exclude / include.